server/src/main/java/org/opensearch/transport/RemoteClusterService.java

```diff
@@ -100,7 +100,6 @@
             Setting.positiveTimeSetting(
                     "cluster.remote.initial_connect_timeout",
                     SEARCH_REMOTE_INITIAL_CONNECTION_TIMEOUT_SETTING, // the default needs to be thirty seconds when fallback is removed
-                    TimeValue.timeValueSeconds(30),
```

Comment:
This says "the value of this setting needs to be at least 30" which is not what it intends to do looks like, so this is just a bug.